vars/fcosKola.groovy

```diff
@@ -46,7 +46,7 @@ def call(params = [:]) {
         }
 
         try {
-            if (params['basicScenarios']) {
+            if (!params['skipBasicScenarios']) {
```

Comment:
We will also need to change other projects using it as cosa:

https://github.com/coreos/coreos-assembler/blob/ee62041f32b8f916a5f7b3b2ac0143de381e9f12/.cci.jenkinsfile#L22

Reply:
https://github.com/coreos/coreos-assembler/pull/2422 covers cosa, and I didn't find any others that enable the old flag.

Reply:
Thanks @bgilbert. I'm not aware of other place, maybe @jlebon or @dustymabe can confirm it.

Reply:
It's in f-c-c too -- I think we'd just need to revert https://github.com/coreos/fedora-coreos-config/commit/82859cbc47bdd0b13d61583431f82fb9456dfb49.

Reply:
Whoops, my branch wasn't updated when I grepped. :+1:

Reply:
https://github.com/coreos/fedora-coreos-config/pull/1209